build/pxc-entrypoint.sh

```diff
@@ -302,8 +302,11 @@ if [ -z "$CLUSTER_JOIN" ] && [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 		fi
 
 		if [ -z "$MYSQL_INITDB_SKIP_TZINFO" ]; then
+		(
+			echo "SET @@SESSION.SQL_LOG_BIN = off;"
```

Comment:
missing pipe?

Reply:
also it's not clear for me, why we're setting this if `MYSQL_INITDB_SKIP_TZINFO` is empty?

Reply:
actually, i've just made the same as there =)
https://github.com/percona/percona-docker/pull/435/files

Reply:
@qJkee but version in https://github.com/percona/percona-docker/pull/435/files and in this PR are very different

Reply:
dc1d727